### ISSUE TITLE
Client/Server Entity Mismatch (See Picture)

### ISSUE THREAD

**Issue by sonicer105** [CoFH/Feedback#880]
Whenever a Blizz, Blitz or Basalz is spawned in the world on a *server*, they appear as an Enderminy, Concussion Creeper, and Fallen Knight to my client and the server still recognizes them correctly. In survival, the Blitz explodes and becomes invisible and impossible to hit under any condition. They appear correctly in single player. Please see this:
![screenshot](https://cloud.githubusercontent.com/assets/6855083/10556829/ded41168-7445-11e5-8f0c-f34c69fdacd0.png)

I am using the Direwolf Modpack 1.7.10 (ver 1.7.0) client and server With forge worldedit added.

**Comment by KingLemming** [comment]
Does it happen with the normal DW20 pack, and does it happen on a new world, or only a particular saved world?

**Comment by sonicer105** [comment]
It exclusively happens on multiplayer servers. It dose not happen in single player and i have not tested it on other servers yet.

**Comment by sonicer105** [comment]
This is an account of what i did. If you just want my findings, skip to the line starting with "tl;dr"

So here's what i did. I backed up and deleted my entire FTB Launcher and re-downloaded the Direwolf20 1.7 pack using the same version as before and created a fresh server using a fresh download. Other then changing the install directory, I have not touched a single setting on the client and server this time around. I started the server.

I was unable to reproduce the result. 

Next, I used my older server i had the problem with and the newly downloaded client to try and reproduce the result.

It worked. The three mobs were displaying incorrectly as the Ender Zoo mobs. That rules out my client as the issue. It must be related to my server setup or that specific world.

As my next test, I backed up that server's world and deleted it. I then started the server again to generate a new one. 

Once again, it worked. That means there must be something on this spicific server set-up that is causing this.

After that, I tried removing both forge world edit and aroma backup mods to see if that made a difference as those were the only ones i added and enabled.

No luck. They STILL displayed incorrectly.

When i made the server initially, I deleted Ender Zoo because i find it irritating, so I restored it to see if it made a difference.

It worked. The mobs are now displaying correctly. This is still problematic because i hate Ender Zoo and don'y want to have it but i can't just tell all the people connecting to the server to delete it. Deleting Ender Zoo on both the client and server also seams to solve the issue but 

tl;dr: It only seams to be an issue if Ender Zoo in installed on the client but not the server.

So, should i report this to the Ender Zoo dev(s) or is this a problem on your end?

**Comment by KingLemming** [comment]
This is an Ender Zoo issue. Basically if it causes that sort of problem when it's only on one side, then it should be required on both client and server. We do this for all of our mods; they should be doing it for theirs.

**Comment by sonicer105** [comment]
I'll make a report on Ender Zoo's bug tracker. Thanks for all the help!
